### PR TITLE
Allow passing None for nullable args

### LINKED ISSUE #59: Creating a C# Nullable Int32 within Python (using Python.NET) to call a C# method with an optional int argument
http://stackoverflow.com/questions/26742837

### PR DESCRIPTION
**Pull request by dhirschfeld**
Fixes #59

### CODE REVIEW

**Comment by dhirschfeld** [comment]
Before this PR:
image
...and after:
image

File: src/runtime/converter.cs [hunk 1]
```diff
@@ -318,6 +318,17 @@ internal static IntPtr ToPythonImplicit(object value)
                 return true;
             }
 
+            if( obType.IsGenericType && obType.GetGenericTypeDefinition() == typeof(Nullable<>) )
```
Comment (filmor) [inline src/runtime/converter.cs]: Please adjust the style here.

**Comment by dhirschfeld** [comment]
@filmor - comments addressed and commit amended.

**Comment by filmor** [comment]
Thank you very much for your contribution :)

**Comment by filmor** [comment]
Ah, sorry, I forgot:

You didn't use the PR-template, so please open another PR in which you
- [ ] Add yourself to AUTHORS
- [ ] Update the CHANGELOG

**Comment by dhirschfeld** [comment]
@filmor, thanks for merging - I think it's a very useful improvement. Changelog & Authors updated in #462

**Comment by davesliu** [comment]
@dhirschfeld  I have a similar issue that for pass null parameter with string type, the code fix in #460 seems not work for it.

**Pull request merged**